ses

```diff
@@ -254,19 +254,21 @@ section_header "Ceph log files"
 
 if [ -d /var/log/ceph ]; then
     mkdir -p $LOG/ceph/log/ceph
-    # Needs to be recursive, as if there's any containerized logs in
-    # files, they're under /var/log/ceph/$FSID/
-    cp -r /var/log/ceph/* $LOG/ceph/log/ceph/
+    # Copy any log files directly in /var/log/ceph (e.g. /var/log/ceph/cephadm.log)
+    cp /var/log/ceph/*.log $LOG/ceph/log/ceph/
+    # Copy any log files in $FSID subdirectories (which is where containerized
+    # daemons may log to, at least for OSD creation and ceph-volume invocations)
+    cp /var/log/ceph/*/*.log $LOG/ceph/log/ceph/
```

Comment:
what about copying compressed logs?

Reply:
Good point, 2dbbf71 should take care of that.